lib/src/driver/driver.dart

```diff
@@ -134,7 +134,7 @@ class BazelWorkerDriver {
   void _runWorker(Process worker, _WorkAttempt attempt) {
     var rescheduled = false;
 
-    runZoned(() async {
+    runZonedGuarded(() async {
```

Comment:
The min SDK needs to be updated for this. I think to `2.8.1`.

Reply:
We didn't add a `@Since?`

Reply:
It does have a `@Since`, I don't know why the analyzer isn't surfacing it...

https://github.com/dart-lang/sdk/blob/546fdf55bccaa02105c3cb0645f2325f5145f5ef/sdk/lib/async/zone.dart#L1588-L1589